### PR TITLE
Add an option for peers to indicate it's an direct session

### PR DESCRIPTION
**Pull request by samip5**

### CODE REVIEW

**Comment by natesales** [comment]
Thanks for the patch! Can we update the description to be more specific? [BIRD's own docs](https://bird.network.cz/?get_doc&v=20&f=bird-6.html#ss6.3) say `Specify that the neighbor is directly connected` to describe the `direct` option.

File: internal/embed/templates/peer.tmpl [hunk 1]
```diff
@@ -27,6 +27,7 @@ protocol bgp {{ UniqueProtocolName $peer.ProtocolName $af }} {
     {{ if StrDeref $peer.Description }}description "{{ StrDeref $peer.Description }}";{{ end }}
     {{ if BoolDeref $peer.Disabled }}disabled;{{ end }}
     {{ if BoolDeref $peer.Passive }}passive;{{ end }}
+    {{ if BoolDeref $peer.Direct }}direct; {{ end }}
```
Comment (natesales) [inline internal/embed/templates/peer.tmpl:30]: Super minor nitpick: there's an extra space here.

**Review by natesales — CHANGES_REQUESTED** [review]

File: internal/embed/templates/peer.tmpl (hunk 1 above)
Comment (samip5) [reply internal/embed/templates/peer.tmpl]: Fixed. :)

**Comment by samip5** [comment]
Could you also release the newer minor version with this?

**Comment by samip5** [comment]
Is the apt repo updated manually or how long after a new release does it appear in apt repo?

**Pull request merged**